Automattic/wp-calypso
# Command Palette: Assume current site for commands

Related to:
- https://github.com/Automattic/dotcom-forge/issues/5842

## Proposed Changes

* We add an injection point to the existing code where, after the normal process, we determine if we should filter and map the siteFunction onclick to a command callback based on if we have a current site or not.

## Testing Instructions



* Use the Calypso Live Link
* Operate with the command palette with a site selected, you should see single site commands.
* Operate with the command palette without a site selected (/sites), you should see multi-site commands.
* All commands should work as expected.
* Check for regressions.

## Pre-merge Checklist
